apps/landing/src/styles/global.css

```
@@ -0,0 +1,221 @@
+}
+
+.hero-panel dt {
+    margin-bottom: 8px;
+    font-size: 0.85rem;
+    font-weight: 700;
+    letter-spacing: 0.04em;
+}
+
+.hero-panel dd {
+    margin: 0;
+    word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)